Package.swift

```diff
@@ -1,30 +1,27 @@
 // swift-tools-version:5.2
-// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "SegmentedPicker",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_15),
+        .watchOS(.v6),
+        .tvOS(.v13)
```

Comment:
```suggestion
```

Reply:
I suggest we remove it then from the supported platforms for now.. 
 Note that the tvOS picker looks like: 

<img width="149" alt="image" src="https://user-images.githubusercontent.com/32329188/80470193-f389df00-8941-11ea-9114-6d095c563ce7.png">

And shouldn't be too hard to support